crates/libwebrtc-sys/Cargo.toml

```diff
@@ -17,11 +18,14 @@ derive_more = { version = "2.0", features = ["deref", "deref_mut", "from"] }
 [build-dependencies]
 anyhow = "1.0"
 cxx-build = "1.0"
+dotenv = "0.15.0"
```

Comment:
Do not add patch versions here. Just use `0.15` the same way as it used everywhere in `Cargo.toml`. We do not specify it, because it can be ignored by `cargo` and writing exact version will confuse reader into thinking that exactly that version is used here (but it is not guaranteed).